### PR TITLE
Add Turkish dictionary

### PR DESCRIPTION
**Pull request by koheiw**
@LungtaSEKI created a Turkish dictionary for newsmap.

### CODE REVIEW

**Comment by koheiw** [comment]
@LungtaSEKI see the comments.

File: dict/turkish.yml [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+    'SO': [Somali, Somalili*, Mogadişu]
+    'TZ': [Tanzanya, Tanzanyalı*, Dodoma, Darelam]
+    'UG': [Uganda, Ugandalı*, Kamp]
+    'YT': [Mayotte, Mahorlu*, Mamoudzou]
+    'ZM': [Zambiya, Zambiyalı*, Lusaka]
+    'ZW': [Zimbabve, Zimbabveli*, Harare]
+
+  MIDDLE:
+    'AO': [Angola, Angolalı*, Luanda]
+    'CD': [Kongo Demokratik Cumhuriyeti, Kongo DC, KDC, Demokratik Kongolu*, DR Kongolu*, Kinşasa]
```
[the remark this answers is not in the document]
Comment (LungtaSEKI) [reply dict/turkish.yml:29]: Thank you for pointing out. I understand that.

File: dict/turkish.yml [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+    'CF': [Orta Afrika Cumhuriyeti, OAC, Orta Afrikalı*, Bangui]
+    'CG': [Kongo, Kongo Cumhuriyeti, Kongolu*, Brazavil]
+    'CM': [Kamerun, Kamerunlu*, Yaounde, Yaoundé]
+    'GA': [Gabon, Gabonlu*, Librevil]
+    'GQ': [Ekvator Ginesi, Ekvator Ginesi Cumhuriyeti, Ekvator Ginesili*, Malabo]
+    'ST': [Sao Tome ve Princ, São Tomé ve Príncipe, São Tomé ve Príncipe, Sao Tome'li*]
```
[the remark this answers is not in the document]
Comment (LungtaSEKI) [reply dict/turkish.yml:35]: Yes it is.

File: dict/turkish.yml [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+    'TD': [Çad, Çadlı*, N'Djamena]
+
+  NORTH:
+    'DZ': [Cezayir, Cezayirli*, Cezayir]
+    'EG': [Mısır, Mısırlı*, Kahire]
+    'EH': [Batı Sahra, Sahralı*, El Aaiun]
+    'LY': [Libya, Libyalı*, Trablus]
+    'MA': [Fas, Faslı*, Rabat]
+    'SD': [Sud, Sudanlı*, Hartum]
```
[the remark this answers is not in the document]
Comment (LungtaSEKI) [reply dict/turkish.yml:44]: Sorry there was a typo. You are right.

File: dict/turkish.yml [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+    'LS': [Lesoto, Lesotolu*, Maseru]
+    'NA': [Namibya, Namibyalı*, Vindhuk]
+    'SZ': [Esvatini, Svazili*, Lobamba, Mbabane]
+    'ZA': [Güney Afrika, G. Afrikal, GA, Güney Afrikalı*, Güney Afrika*, Cape Town, Johannesburg, Pretoria]
+
+  WEST:
+    'BF': [Burkina Faso, Burkinalı*, Vagadugu]
+    'BJ': [Benin, Beninli*, Porto Novo]
+    'CI': [Fildişi Sahili, Fildişi Sahili Cumhuriyeti, Côte d'Ivoire, Fildişi Sahilli*, Yamusukro, Abican]
+    'CV': [Yeşil Burun Adaları, Yeşilunlu*, Praia]
```
[the remark this answers is not in the document]
Comment (LungtaSEKI) [reply dict/turkish.yml:59]: Thank you for your comment. I agree with you. It was my mistake.

File: dict/turkish.yml [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+    'SE': [İsveç, İsveçli*, Stockholm]
+    'SJ': [Svalbard ve Jan Mayen Adaları]
+
+  SOUNTH:
+    'AD': [Andorra, Andorralı*]
+    'AL': [Arnavutluk, Arnavut*, Tiran]
+    'BA': [Bosna, Bosnalı*, Bosna Hersek, Hersek, Sarbosna]
+    'ES': [İspanya, İspanyalı*, Madrid, Barselona]
+    'GI': [Cebelitarık, Cebelitarıklı*, Llanitos]
+    'GR': [Yunanistan, Yunan, Yunanlı*, yunanistanlı*, Atina]
```
[the remark this answers is not in the document]
Comment (LungtaSEKI) [reply dict/turkish.yml:237]: I think I made another typo here too. I'm sorry but thank you for finding it out.

File: dict/turkish.yml [hunk 6]
```diff
@@ -0,0 +1,294 @@
...
+    'HR': [Hırvatistan Hırvat*, Hırvatistanlı*, Zagreb]
+    'IT': [İtalya, İtalylı*, Roma]
```
[the remark this answers is not in the document]
Comment (LungtaSEKI) [reply dict/turkish.yml:239]: Thank you for your comment. I did not know that.

File: dict/turkish.yml [hunk 7]
```diff
@@ -0,0 +1,294 @@
...
+    'SM': [San Marino, San Marinolu*]
+    'VA': [Vikan]
+
+  WEST:
+    'AT': [Avusturya, Avusturyalı*, Viyana]
+    'BE': [Belçika, Belçikalı*, Brüksel]
+    'CH': [İsviçre, İsviçreli*, Zürih, Bern]
+    'DE': [Almanya, Alman*, Berlin, Frankfurt]
+    'FR': [Fransa, Fransız*, Paris]
+    'LI': [Liechtenstein, Liechtensteinli*, Vaduz]
```
[the remark this answers is not in the document]
Comment (LungtaSEKI) [reply dict/turkish.yml:256]: I think I made a grammatical error here. Thank you for letting me know.

File: dict/turkish.yml [hunk 8]
```diff
@@ -0,0 +1,294 @@
...
+    'YE': [Yemen, Yemenli*, Sanaa]
+
+EUROPE:
+  EAST:
+    'BG': [Bulgaristan, Bulgar*, Sofya]
+    'BY': [Belarus, Belaruslu*, Minsk]
+    'CZ': [Çek Cumhuriyeti, Çekya, Çek*, Prag] # Çekya
+    'HU': [Macaristan, Macar Cumhuriyeti, Macar*, Budapeşte]
+    'MD': [Moldova, Moldovyalı*, Kişinev]
+    'PL': [Polonya, Polonyalı*, Varşova]
```
[the remark this answers is not in the document]
Comment (LungtaSEKI) [reply dict/turkish.yml:207]: Should we add Leh as well then?

File: dict/turkish.yml [hunk 9]
```diff
@@ -0,0 +1,294 @@
...
+  WEST:
+    'AT': [Avusturya, Avusturyalı*, Viyana]
+    'BE': [Belçika, Belçikalı*, Brüksel]
+    'CH': [İsviçre, İsviçreli*, Zürih, Bern]
+    'DE': [Almanya, Alman*, Berlin, Frankfurt]
+    'FR': [Fransa, Fransız*, Paris]
+    'LI': [Liechtenstein, Liechtensteinli*, Vaduz]
+    'LU': [Lüksemburg, Lüksemburglu*]
+    'MC': [Monako, Monakolu*]
+    'NL': [Hollanda, Hollandalı*, Amsterdam]
```
[the remark this answers is not in the document]
Comment (LungtaSEKI) [reply dict/turkish.yml:259]: Thank you. We can also add Felemenk here.

**Pull request merged**